ui/selectors/selectors.js

```diff
@@ -233,6 +234,44 @@ export function getHardwareWalletType(state) {
   return isHardwareWallet(state) ? keyring.type : undefined;
 }
 
+/**
+ * Returns the hardware wallet device for the current wallet
+ *
+ * @param {object} state
+ * @returns {string | undefined}
+ */
+export function getHardwareWalletDevice(state) {
```

Comment:
It doesn't do the exact same thing, but there is another selector for `getDeviceModel` that we might be able to reuse here?

Reply:
Thanks, yeah it looks like `getDeviceModel` doesn't do the direct mapping and also requires a backend message call.